src/beanmachine/tutorials/utils/plots.py

```diff
@@ -2,43 +2,85 @@
 #
 # This source code is licensed under the MIT license found in the
 # LICENSE file in the root directory of this source tree.
-
```

Comment:
Our internal linter is complaining about the lack of a new line after the comments at the beginning of the file (before the docstring) :)